src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/DecisionOutcomeDashboardHandler.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.callback.DashboardCallbackHandler;
+import uk.gov.hmcts.reform.civil.client.DashboardApiClient;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.DashboardNotificationsParamsMapper;
+import uk.gov.hmcts.reform.civil.service.FeatureToggleService;
+
+import java.util.List;
+
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.CREATE_DASHBOARD_NOTIFICATION_DECISION_OUTCOME;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.DashboardScenarios.SCENARIO_AAA6_DECISION_OUTCOME;
+
+@Service
+public class DecisionOutcomeDashboardHandler extends DashboardCallbackHandler {
```

Comment:
Please create a claimant and defendant handler - because otherwise it might update the database even if claimant or defendant is LR, but we don't want anything in the database for LR.